public/public.php

```diff
@@ -472,12 +472,12 @@ public function list_post_attachments_with_sources( $post_id = 0 ) {
 			ISC_Log::log( sprintf( 'going through %d attachments', count( $attachments ) ) );
 			$atts = [];
 			foreach ( $attachments as $attachment_id => $attachment_array ) {
-				$use_standard_source = Standard_Source::use_standard_source( $attachment_id );
-				$source              = self::get_image_source_text( $attachment_id );
+				$image_uses_standard_source = Standard_Source::use_standard_source( $attachment_id );
+				$source                     = self::get_image_source_text( $attachment_id );
 
 				// check if source of own images can be displayed
-				if ( ( $use_standard_source === '' && $source === '' ) || ( $use_standard_source !== '' && $exclude_standard ) ) {
-					if ( $use_standard_source !== '' && $exclude_standard ) {
+				if ( ( ! $image_uses_standard_source && $source === '' ) || ( $image_uses_standard_source && $exclude_standard ) ) {
+					if ( $image_uses_standard_source && $exclude_standard ) {
 						ISC_Log::log( sprintf( 'image %d: "own" sources are excluded', $attachment_id ) );
 					} else {
 						ISC_Log::log( sprintf( 'image %d: skipped because of empty source', $attachment_id ) );
```

Comment:
**Suggestion:** Simplify the conditional logic in the `if` statement at line 479. You can combine the two conditions into one by using the logical OR operator (`||`). [maintainability]
```suggestion
    if ( ! $image_uses_standard_source && $source === '' || $image_uses_standard_source && $exclude_standard ) {
        if ( $image_uses_standard_source && $exclude_standard ) {
            ISC_Log::log( sprintf( 'image %d: "own" sources are excluded', $attachment_id ) );
        } else {
            ISC_Log::log( sprintf( 'image %d: skipped because of empty source', $attachment_id ) );
        }
    }
```